presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -61,17 +64,20 @@
     public synchronized Response install(@NotNull @ApiParam("Url to fetch package") String locationToFetchPackage,
             @QueryParam("checkDependencies") @DefaultValue("true") @ApiParam("If false, disables dependency checking") boolean checkDependencies)
     {
+        LOGGER.debug("PUT /package  url: {} checkDependencies: {}", locationToFetchPackage, checkDependencies);
         try {
             installedPackageType = PackageType.valueOf(getFileExtension(locationToFetchPackage).toUpperCase());
         }
         catch (IllegalArgumentException e) {
+            LOGGER.error("Invalid package type for presto-server. Package must be .tar.gz or .rpm", e);
```

Comment:
I would include the URL here, so the invalid package can be diagnosed.

"Invalid package '[URL]'; expected '.tar' or '.rpm'."